core/trino-main/src/main/java/io/trino/execution/scheduler/TopologyAwareNodeSelector.java

```diff
@@ -61,7 +61,7 @@
     private final AtomicReference<Supplier<NodeMap>> nodeMap;
     private final int minCandidates;
     private final long maxSplitsWeightPerNode;
-    private final long maxPendingSplitsWeightPerTask;
```

Comment:
`TopologyAwareNodeSelector` should still use `maxPendingSplitsWeightPerTask` for backward compatibility

Reply:
Scratch that, see my other comment: https://github.com/trinodb/trino/pull/15168#discussion_r1049628263